package com.desafio.desafio3.Exceptions.ExceptionsKinds;

public class UserNotFoundException extends RuntimeException{
    public UserNotFoundException(String message){ super(message);}
}
